docs/keosd/basic-usage.md

```diff
@@ -0,0 +1,109 @@
+For most users the easiest way to use keosd is to have cleos launch it automatically. Wallet files will also be created in the default directory (`~/eosio-wallet`).
+
+## Launching keosd manually
+
+It is possible to launch `keosd` manually simply with
+
+```
+$ keosd
+```
+
+By default, `keosd` creates the folder `~/eosio-wallet` and populates it with a basic `config.ini` file.  The location of the config file can be specified on the command line using the `--config-dir` argument.  The configuration file contains the http server endpoint for incoming http connections and other parameters for cross origin resource sharing.
+
+The location of the wallet data folder can be specified on the command line using the --data-dir argument.
+
+## Auto locking
+
+By default, `keosd` is set to lock your wallets after 15 minutes of inactivity. This is configurable in the `config.ini` by setting the timeout seconds in `unlock-timeout`.
+
+Be aware if you need to disable this feature you will have to set an enormous number -- setting it to 0 will cause keosd to always lock your wallet.
+
+## Stopping keosd
+
+The most effective way to stop `keosd` is to find the keosd process and send a SIGTERM signal to it.
+
+## Other all options
+
+For a list of all commands known to `keosd`, simply run it with no arguments:
+
+```bash
```

Comment:
Is this section auto generated from the actual output of the command?  it seems highly likely to drift as we add things to the options

Reply:
It is not auto-generated at the moment. I think it is a good idea to programmatically do this though

Reply:
@b1bart As discussed last January, a custom parser would have to be written and maintained for cleos/keosd cli's. This is absolutely the mid-longterm goal.